README.md

```diff
@@ -1,8 +1,7 @@
-# BBC Accessibility Standards Checker
+# BBC Accessibility Guidelines Checker
 
 This tool runs a set of tests against a set of URLs to verify whether each one
-meets the [BBC accessibility
-standards](http://www.bbc.co.uk/guidelines/futuremedia/accessibility/).
+meets the [BBC accessibility guidelines](http://www.bbc.co.uk/guidelines/futuremedia/accessibility/).
```

Comment:
This URL will need to be updated fairly soon ... perhaps best to leave this PR until then, and do a full sweep of the documentation at the same time.